3des/3des.c

```diff
@@ -0,0 +1,61 @@
+#include <openssl/des.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+int main()
+{
+    DES_cblock cb1 = {0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE};
+    DES_cblock cb2 = {0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE};
+    DES_cblock cb3 = {0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE, 0xAE};
+
+    DES_key_schedule ks1, ks2, ks3;
+
+    DES_cblock cblock = {0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+
+    char string[] = "I am a software developer";
```

Comment:
Please take plaintext input as CLI argument.
nit: Also, rename `string` to `plaintext` maybe?